ch05.adoc

```diff
@@ -747,3 +747,291 @@ data:
 
 In this example both the analysis time and the single pressure level are represented using scalar coordinate variables. The analysis time is identified by the standard name "forecast_reference_time" while the valid time of the forecast is identified by the standard name "time".
 
+
+[[domain-variables, Section 5.8, "Domain Variables"]]
+=== Domain Variables
+
+A domain describes data locations and cell properties. It defines
+cells that span a collection of dimensions with cell coordinates, cell
+measures, and coordinate reference systems.
+
+A data variable defines its domain via its own attributes, but a
+domain variable provides the description of a domain in the absence of
+any data values. It is of arbitrary type since it contains no data. It
+acts as a container for the attributes that define the domain. The
+purpose of a domain variable is to provide domain information to
+applications that have no need of data values at the domain's
+locations, thus removing any ambiguity when retrieving a domain from a
+dataset. Ancillary variables and cell methods are not part of the
+domain, because they are only defined in relation to data values.
+
+The domain variable supports the same attributes as are allowed on a
+data variable for describing a domain, with exactly the same meanings
+and syntaxes, as described in <<attribute-appendix>>. If an attribute
+is needed by a particular data variable to describe its domain, then
+that attribute would also be needed by the equivalent domain variable.
+
+The dimensions of the domain must be stored with the **`dimensions`**
+attribute. It is necessary to list these dimensions, rather than
+inferring them from the contents of the other attributes, as it can
+not be guaranteed that the referenced variables span all of the
+required dimensions (as could be the case for a discrete axis, for
+instance). The value of the **`dimensions`** attribute is a blank
+separated list of the dimensions names. There is no restriction on the
+order in which the dimensions appear in the **`dimensions`** attribute
+string. The presence of a **`dimensions`** attribute will identify the
+variable as a domain variable. If a domain has no dimensions then the
+value of the **`dimensions`** attribute must be an empty string.
```

Comment:
Can a domain exist without any dimensions?

Reply:
(I should have read the existing text first rather than replying from memory - sorry restated a lot of stuff uncontested stuff. I'll try again)

A domain can certainly exist without _explicit_ dimensions, as they may be implied by scalar coordinate variables.

Would this be valid domain, though?
```
    char domain ;
        domain:dimensions = "" ;
        domain:grid_mapping = "lambert_conformal" ;
```
My gut feeling is "yes" ....